<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>847</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_main_window">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>541</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>840</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main View</string>
     </attribute>
     <widget class="QListWidget" name="event_list_area">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>731</width>
        <height>441</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>730</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>5</y>
        <width>831</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>790</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Upcoming Charity Events</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Account details</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>My Enrolled Charities</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>My Enrolled Events</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_ad_charities">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="list_widget_ad_events">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_ad_user_id">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>60</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_ad_password">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_change_user_id">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Change Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_cancel_changes">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:lightgrey; border: 1px solid grey; border-radius: 5px; margin-top: 1px;</string>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_withdraw_event">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>430</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Withdraw from event</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_change_password">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_confirm_changes">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Confirm Changes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ad_remove_account">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>811</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:lightgrey; border: 1px solid grey; border-radius: 5px; margin-top: 1px;</string>
      </property>
      <property name="text">
       <string>Remove Account</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ad_unavailable_user_id">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ad_unauthorized_password">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage Account Details</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>791</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage Event Enrollments</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>log out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
